tests/dummy/app/routes/signin.js

```diff
@@ -26,6 +26,7 @@ export default Route.extend({
           this.controller.transitionToRoute('secure');
         })
         .catch((err) => {
+          debugger;
```

Comment:
Probably want to lose this and its other debugger friends.

Reply:
all that's in the dummy app :)